numpy/testing/tests/test_utils.py

```diff
@@ -89,16 +89,27 @@ def foo(t):
     def test_0_ndim_array(self):
         x = np.array(473963742225900817127911193656584771)
         y = np.array(18535119325151578301457182298393896)
-        assert_raises(AssertionError, self._assert_func, x, y)
+        
+        with pytest.raises(AssertionError) as exc_info:
+            self._assert_func(x, y)
+        msg = str(exc_info.value)
+        assert_('Mismatched elements: 1 / 1 (100%)\n'
+        in msg)
```

Comment:
nit, here and below: you can pass a `match=some_regex` to `pytest.raises` to check for a matching message.

Reply:
Thank you for the review, i have updated the error message checks to use match